locales-pending/landing-premium.ftl

```diff
@@ -12,3 +12,62 @@ landing-premium-hero-image-removal-badge-label = exposures auto‑removed
 
 # This is a label underneath a big number "50" - it's an image that demos Monitor.
 landing-premium-hero-image-chart-label = fixed
+
+landing-premium-plans-heading = Choose your level of protection
+landing-premium-plans-lead = We believe in your right to privacy, so data breach monitoring is always free. For more robust protection, { -brand-monitor } also offers continuous automatic removal of your personal information.
+landing-premium-plans-table-annotation-plus = Recommended
+landing-premium-plans-table-heading-feature = Features
+# Users can click a button with this label to see a popover with a bit more detail.
+landing-premium-plans-table-feature-callout-trigger = More info
+landing-premium-plans-table-heading-free-title = { -brand-monitor }
+landing-premium-plans-table-heading-free-subtitle = Free breach alerts
+landing-premium-plans-table-heading-plus-title = { -brand-monitor } <b>Plus</b>
+landing-premium-plans-table-heading-plus-subtitle = Automatic data removal
+# Variables:
+#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
+landing-premium-plans-table-feature-scan-label = Scan { $dataBrokerTotalCount } data broker sites that may be selling your personal info
+landing-premium-plans-table-feature-scan-free = One-time
+landing-premium-plans-table-feature-scan-plus = Monthly
+landing-premium-plans-table-feature-removal-label = Remove personal info from sites that are selling it
+landing-premium-plans-table-feature-removal-free = Manual removal
+landing-premium-plans-table-feature-removal-plus = Automatic removal
+landing-premium-plans-table-feature-removal-free-callout = We’ll let you know which data brokers are selling your info so you can contact them to request removal.
+# Variables:
+#   $dataBrokerTotalCount (number) - number of scanned data broker sites, e.g. 190
+landing-premium-plans-table-feature-removal-plus-callout = We’ll automatically request removal of your private info across more than { $dataBrokerTotalCount } data broker sites.
+landing-premium-plans-table-feature-alerts-label = Get alerts when your data has been breached
+landing-premium-plans-table-feature-alerts-free = Included
+landing-premium-plans-table-feature-alerts-plus = Included
+landing-premium-plans-table-feature-guidance-label = Fix high-risk data breaches
+landing-premium-plans-table-feature-guidance-free = Guided
+landing-premium-plans-table-feature-guidance-plus = Guided
+landing-premium-plans-table-feature-monitoring-label = Continuous monitoring
+landing-premium-plans-table-feature-monitoring-free = Included
+landing-premium-plans-table-feature-monitoring-plus = Included
+landing-premium-plans-table-billing-label = Price
+landing-premium-plans-table-billing-free = Always free
+# Label for the options "Yearly" and "Monthly"
+landing-premium-plans-table-billing-plus-period-label = Billing
+landing-premium-plans-table-billing-plus-period-annually = Yearly
```

Comment:
Should we change the id to `*-yearly` to match the value?

Reply:
I initially aligned with Relay, but internal consistency makes more sense: 8316eed0d